tsconfig.json

```diff
@@ -7,11 +7,8 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es2018",
```

Comment:
you changed this to es2020 in the other Angular 11 upgrades. Should it be es2020 here too?